prov/efa/docs/lttng.md

```diff
@@ -17,6 +17,8 @@ If these are missing, they should be fine to install via `yum` or other equivale
 * [libuuid](https://sourceforge.net/projects/libuuid/)
 * [popt](https://directory.fsf.org/wiki/Popt)
 * [libxml2](http://www.xmlsoft.org/)
+* [glib2-devel] (glibc 2.28 is required for babeltrace2)
+* [elfutils-devel] (required for babeltrace2)
```

Comment:
- Provide a link or remove the link formatting.
- Use the generic names for the packages. `glib2-devel` and `elfutils-devel` are specific to `yum`.
- Babeltrace2's dependency is pedantically GLib, not glibc ([ref](https://github.com/efficios/babeltrace/blob/stable-2.0/README.adoc#build-time-requirements))
- I would also recommend moving these to the section on Babeltrace2, since it's not a hard requirement for LTTng-UST (i.e. you can analyze LTTng traces using other tools, like Trace Compass)

```suggestion
* [GLib](https://developer.gnome.org/glib/) >= 2.28
* [elfutils](https://sourceware.org/elfutils/) >= 0.154
```